backend/src/main/java/gov/cdc/usds/simplereport/api/pxp/PatientRegistrationController.java

```diff
@@ -0,0 +1,89 @@
+package gov.cdc.usds.simplereport.api.pxp;
+
+import gov.cdc.usds.simplereport.db.model.PatientRegistrationLink;
+import gov.cdc.usds.simplereport.db.model.Person;
+import gov.cdc.usds.simplereport.db.model.auxiliary.PatientRegistration;
+import gov.cdc.usds.simplereport.service.PatientRegistrationLinkService;
+import gov.cdc.usds.simplereport.service.PersonService;
+import java.util.UUID;
+import javax.servlet.http.HttpServletRequest;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.security.access.prepost.PostAuthorize;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping("/pxp/register")
+@PreAuthorize("@patientRegistrationLinkService.flagRegistrationRequest()")
+@PostAuthorize("@restAuditLogManager.logRestSuccess(#request, returnObject)")
+@Validated
+public class PatientRegistrationController {
+  private static final Logger LOG = LoggerFactory.getLogger(PatientRegistrationController.class);
+
+  private final PatientRegistrationLinkService _patientRegLinkService;
+  private final PersonService _personService;
+  private final CurrentPatientContextHolder _currentPatientContextHolder;
+
+  public PatientRegistrationController(
+      PersonService personService,
+      PatientRegistrationLinkService patientRegistrationLinkService,
+      CurrentPatientContextHolder currentPatientContextHolder) {
+    _personService = personService;
+    _patientRegLinkService = patientRegistrationLinkService;
+    _currentPatientContextHolder = currentPatientContextHolder;
+  }
+
+  @PostMapping("/")
```

Comment:
I rebased my branch onto this one and got a 404 posting to this endpoint until I changed this to `@PostMapping("")`

Reply:
Whoof I _just_ squash/rebased this onto main – funny as I had "" originally but changed it bc I felt that "/" was more pendantic. Will fix!

Reply:
Done in f98a396